docs/events/indicators-of-compromise.asciidoc

```diff
@@ -0,0 +1,95 @@
+[[indicators-of-compromise]]
+= Indicators of compromise
+
+The Indicators page collects data from enabled threat intelligence feeds and provides a centralized view of indicators, also known as indicators of compromise (IoCs). This topic helps you set up the Indicators page  and explains how to work with IoCs.
```

Comment:
I would refer to Intelligence page when talking about the specific nav menu item. The Intelligence menu item in the future is planned to be the home of multiple different capabilities related to Threat Intelligence. When we talk about the technical specifics of the only available page atm which is Indicators page, I'd refer to it as Indicators page

Reply:
Thanks for explaining, @maxcold! That will help us plan for the future.

@nastasha-solomon I'd still say we need to explicitly tell readers how to find the "Indicators" page in the side nav, since we currently don't. We can revise in the future when new pages get built in the "Intelligence" section.

Reply:
Makes sense. I'll add the nav instructions in where needed. Thanks, both!